lib/Agrammon/Email.pm6

```diff
@@ -0,0 +1,76 @@
+use v6;
+use Net::SMTP::Client::Async;
+use MIME::Base64;
+use Email::MIME;
+
+class Agrammon::Email {
+    has $!msg;
+    has $!subject;
+    has $!to;
+    has $!from;
+    has $!attachment;
+#    has $!filename;
+    has $!mail;
+
+    submethod TWEAK( :$attachment, :$msg, :$filename, :$to, :$from, :$subject --> Nil) {
+        $!to = $to;
+        $!from = $from;
+        $!subject = $subject;
```

Comment:
Can do `:$!to, :$!from, :$!subject` in the signature and save these three lines.

Reply:
Done.